packages/myst-to-jats/src/index.ts

```diff
@@ -520,23 +561,29 @@ class JatsSerializer implements IJatsSerializer {
     this.handlers = opts?.handlers ?? handlers;
     this.mdast = copyNode(mdast);
     if (opts?.extractAbstract) {
-      const abstractMdast = extractPart(this.mdast, 'abstract');
-      if (abstractMdast) {
-        const abstractSerializer = new JatsSerializer(this.file, abstractMdast as Root, {
-          isNotebookArticleRep: this.data.isNotebookArticleRep,
-          slug: this.data.slug,
-          handlers: this.handlers,
-        });
-        abstractSerializer.render();
-        this.data.abstract = abstractSerializer.elements();
-      }
+      const abstractParts = opts.abstractParts ?? [{ part: 'abstract' }];
+      this.data.abstracts = abstractParts
+        .map((def) => renderAbstract(this.file, this.mdast, def, opts))
+        .filter((e) => !!e) as Element[];
     }
+    this.data.acknowledgments = renderAcknowledgments(this.file, this.mdast, opts);
+    const backSections = opts?.backSections ?? [
+      {
+        part: ['data-availability', 'data_availability', 'availability'],
```

Comment:
Should this just go in `myst-cli` too, with the various abstract sections? Thinking ahead to making this more extensible, seems like we should have the "myst defaults" in only one place (either here or myst-cli).

Reply:
Yep, good point -- I will move it up to the CLI for now.